nnvm/include/nnvm/c_api.h

```diff
@@ -16,7 +16,8 @@
 #define NNVM_DLL __declspec(dllimport)
 #endif
 #else
-#define NNVM_DLL
+// The NNVM_DLL tag cannot appear right after doxygen comments, else doxygen parser is confused.
+#define NNVM_DLL __attribute__((visibility("default")))
```

Comment:
Could we add `__GNUC__ >=4` guard? please see: https://gcc.gnu.org/wiki/Visibility example